### ISSUE TITLE
The effectiveness of developers

### ISSUE THREAD

**Issue by zhyjljj** [Electry/VitaGrafix#105]
Gravity Rush HD http://tieba.baidu.com/p/5890661615

Resident Evil Revelations 2HD
http://tieba.baidu.com/p/5887568570

**Comment by zhyjljj** [comment]
![5ea8945b444a6f060d8c79f5d116803c](https://user-images.githubusercontent.com/43905636/46571607-5572a480-c9aa-11e8-9ca3-58e8c17ece34.jpg)


![Uploading 5EA8945B444A6F060D8C79F5D116803C.jpg…]()

**Comment by ChiSys** [comment]
would be nice to have some non chinese download links. cant get any further

**Comment by zhyjljj** [comment]
Contact developers and add them to the plug-ins collection

**Comment by zhyjljj** [comment]
Both games are available in Hong Kong, in both Chinese and English

**Comment by TheMostEve** [comment]
How about eboot for US and EU version?

**Comment by Wontell** [comment]
> 
> 
> Gravity Rush HD http://tieba.baidu.com/p/5890661615
> 
> Resident Evil Revelations 2HD
> http://tieba.baidu.com/p/5887568570

I dont understand why they did it for RER2? This game even on fully overclocked Vita has HUGE FPS drops.

**Comment by zhyjljj** [comment]
Because it won't get any worse

**Comment by Babbosh** [comment]
Yep please, someone post both the eboots here. I can't get it to download on Baidu. Strangely enough I landed in Changi 2 days ago!

**Comment by zhyjljj** [comment]
I WILL try ,there is 2 different files,maidumptool load type0 &type5

**Comment by zhyjljj** [comment]
标题：吧友需要的启示录1.04高清EBOOT（加载方式5）
地址：http://tieba.baidu.com/p/5919725116

**Comment by zhyjljj** [comment]
match PCSH00187

**Comment by zhyjljj** [comment]
onedrive https://1drv.ms/u/s!And3_Xvlh7angQWCq1KVDJ34JY8u

**Comment by zhyjljj** [comment]
> 嗨
> 有人可以为REV2上传544p eboot吗？
> 我无法下载它，如果有人能够下载，那就太好了

https://1drv.ms/u/s!And3_Xvlh7angQWCq1KVDJ34JY8u

**Comment by zhyjljj** [comment]
> Yep please, someone post both the eboots here. I can't get it to download on Baidu. Strangely enough I landed in Changi 2 days ago!

where is changi😂😂

**Comment by Babbosh** [comment]
> changi

I meant Lantau, in Changi I landed a the previous week. Ahhahaha

**Comment by Babbosh** [comment]
Will this work with the EU or the US version or do you need the PCSH00187 only?  I can't find it on the PKGJ. There is a Chinese version but it has a different ID.

**Comment by zhyjljj** [comment]
> Will this work with the EU or the US version or do you need the PCSH00187 only? I can't find it on the PKGJ. There is a Chinese version but it has a different ID.

only PCSH00187 ，HongKong

**Comment by zhyjljj** [comment]
> > Will this work with the EU or the US version or do you need the PCSH00187 only? I can't find it on the PKGJ. There is a Chinese version but it has a different ID.
> 
> only PCSH00187 ，HongKong

![screenshot_uc _20181027-081202](https://user-images.githubusercontent.com/43905636/47597293-112e5f00-d9c0-11e8-870f-c47257da1d22.png)

**Comment by zhyjljj** [comment]
> Will this work with the EU or the US version or do you need the PCSH00187 only? I can't find it on the PKGJ. There is a Chinese version but it has a different ID.

if your game come from pkgj, it is will not work.must install with maidumptool &must be ver1.04.

**Comment by Babbosh** [comment]
> if your game come from pkgj, it is will not work.must install with maidumptool &must be ver1.04.

Can you upload the H.K. game somewhere? I can't find it.

**Comment by zhyjljj** [comment]
> > if your game come from pkgj, it is will not work.must install with maidumptool &must be ver1.04.
> 
> Can you upload the H.K. game somewhere? I can't find it.

i can't do that,I do this all by  cellphone&LTE,the vpk too big,3.5GB，can't upload. 
can you download this app?
 http://pandownload.com/2.0.4 
open it,then copy these letters“https://pan.baidu.com/s/1ghkHfeFRgHcLnYRfUdAw5g  提取码：9lPK”, download will auto start
i don't kown will or will not work. I have never been abroad.maybe Baidu SkyDrive can not be used abroad.
Because the law is not perfect.These resources are very easy to download in China.

**Comment by zhyjljj** [comment]
> > if your game come from pkgj, it is will not work.must install with maidumptool &must be ver1.04.
> 
> Can you upload the H.K. game somewhere? I can't find it.

Contact Developer,Ask him to add these games.After all, the games have worked out, and they can be easily added to vitagrafix
@yeqing1990 He's the best and most productive of us

**Comment by Babbosh** [comment]
i can't do that,I do this all by cellphone&LTE,the vpk too big,3.5GB，can't upload.
> can you download this app?
> http://pandownload.com/2.0.4
> open it,then copy these letters“https://pan.baidu.com/s/1ghkHfeFRgHcLnYRfUdAw5g 提取码：9lPK”, download will auto start
> i don't kown will or will not work. I have never been abroad.maybe Baidu SkyDrive can not be used abroad.
> Because the law is not perfect.These resources are very easy to download in China.

I downloaded the app, but it's in Chinese , I can't understand anything. From the main page, it looks like you need an account and therefore need to register. 
Too complicated, I can't read Chinese.

**Comment by zhyjljj** [comment]
> i can't do that,I do this all by cellphone&LTE,the vpk too big,3.5GB，can't upload.
> 
> > can you download this app?
> > http://pandownload.com/2.0.4
> > open it,then copy these letters“https://pan.baidu.com/s/1ghkHfeFRgHcLnYRfUdAw5g 提取码：9lPK”, download will auto start
> > i don't kown will or will not work. I have never been abroad.maybe Baidu SkyDrive can not be used abroad.
> > Because the law is not perfect.These resources are very easy to download in China.
> 
> I downloaded the app, but it's in Chinese , I can't understand anything. From the main page, it looks like you need an account and therefore need to register.
> Too complicated, I can't read Chinese.

NO need login,just open it.copy this “https://pan.baidu.com/s/1ghkHfeFRgHcLnYRfUdAw5g  提取码：9lPK”，download will auto start.or it is not work out of china.

**Comment by Babbosh** [comment]
> NO need login,just open it.copy this “https://pan.baidu.com/s/1ghkHfeFRgHcLnYRfUdAw5g 提取码：9lPK”，download will auto start.or it is not work out of china.

Yes, you need to login. It asks you to download the network disk app and then it's all in Chinese from there.  A google drive upload will help a lot.

**Comment by zhyjljj** [comment]
> > NO need login,just open it.copy this “https://pan.baidu.com/s/1ghkHfeFRgHcLnYRfUdAw5g 提取码：9lPK”，download will auto start.or it is not work out of china.
> 
> Yes, you need to login. It asks you to download the network disk app and then it's all in Chinese from there. A google drive upload will help a lot.

Although there is a login box, but do not need to login at all, I do not login in China, because the download speed will be limited after login.
Just open it and copy that link, and it will be downloaded automatically.

**Comment by zhyjljj** [comment]
> > NO need login,just open it.copy this “https://pan.baidu.com/s/1ghkHfeFRgHcLnYRfUdAw5g 提取码：9lPK”，download will auto start.or it is not work out of china.
> 
> Yes, you need to login. It asks you to download the network disk app and then it's all in Chinese from there. A google drive upload will help a lot.

I do not have any ssr ,can't upload to Google drive

**Comment by yeqing1990** [comment]
Or you put your eboot to me, I change to you?

**Comment by zhyjljj** [comment]
> > NO need login,just open it.copy this “https://pan.baidu.com/s/1ghkHfeFRgHcLnYRfUdAw5g 提取码：9lPK”，download will auto start.or it is not work out of china.
> 
> Yes, you need to login. It asks you to download the network disk app and then it's all in Chinese from there. A google drive upload will help a lot.

@Babbosh 
yeqing1990 commented 14 hours ago
Or you put your eboot to me, I change to you?
--

**Comment by Babbosh** [comment]
Hey guys, sorry for the delay.  I have zipped both Gravity Rush PCSF00024  and REV PCSF00728.
Thanks a lot!!!!

[EBoots.zip](https://github.com/Electry/VitaGrafix/files/2530701/EBoots.zip)

**Comment by zhyjljj** [comment]
> 嘿伙计们，抱歉拖延。我已经压缩了Gravity Rush PCSF00024和REV PCSF00728。
> 非常感谢！！！！
> 
> [EBoots.zip](https://github.com/Electry/VitaGrafix/files/2530701/EBoots.zip)

@yeqing1990

**Comment by yeqing1990** [comment]
Ok, I changed it here at night.

**Comment by zhyjljj** [comment]
> Hey guys, sorry for the delay. I have zipped both Gravity Rush PCSF00024 and REV PCSF00728.
> Thanks a lot!!!!
> 
> [EBoots.zip](https://github.com/Electry/VitaGrafix/files/2530701/EBoots.zip)

tonight,now time is 14:08

**Comment by yeqing1990** [comment]
@Babbosh 
1. The EBOOT file of your PCSF00728 is incorrect. Please directly give me the MAI version of the EBOOT.BIN file.
2. PCSF00024 is not yet capable of HD.

**Comment by yeqing1990** [comment]
@Babbosh
You can use the FAG Dec to decrypt the EBOOT. Give me the file again, then I will send it to you. You can use rePatch to load the modified EBOOT file:
I have given the resolution address of the game to Electry, I hope he can join the next version of VitaGrafix.

**Comment by Babbosh** [comment]
I don't have the mai version. That is the one I have.

**Comment by yeqing1990** [comment]
@Babbosh
#132
